testsuite/tests/ui/test_unique_invoice_id.py

```diff
@@ -0,0 +1,65 @@
+"""
+rewrite of /spec/ui_specs/billing/unique_invoice_id_spec.rb
+For testing this issue, we don't need to use UI
```

Comment:
Could you please include more description?

Reply:
Yes, is it better now?